components/server/package.json

```diff
@@ -18,9 +18,9 @@
     "clean:node": "rimraf node_modules",
     "purge": "yarn clean && yarn clean:node && yarn run rimraf yarn.lock",
     "lint": "yarn tslint -p .",
-    "test": "mocha --opts mocha.opts './**/*.spec.ts' --exclude './node_modules/**'",
-    "db-test": "r(){ . $(leeway run components/gitpod-db:db-test-env); yarn db-test-run; };r",
-    "db-test-run": "mocha --opts mocha.opts '**/*.spec.db.ts' --exclude './node_modules/**'",
+    "test": "yarn db-test",
```

Comment:
I really like to keep these separate, to be able to handle cases where we don't have a DB around/where we want to keep iterations quick. CI (should) calls both anyway.

All ears for a 3rd target `yarn all-tests`. Or if we want to change our muscle memory: `yarn unit-test` or so.

Reply:
Why? Starting the db is reasonably quick, and it avoids a split brain scenario where I have to remember what to call. 

If we want to retain that, then let's make `yarn test` work as it should - runs everything, with somethign like `yarn run test-no-db` being the one without the db setup.

But in practice, I haven't found this to be costly really as the db setup is paid for once only

Reply:
Right, but that's unrelated to what we're doing here. Yes, we need to fix it (and looks like we're using an older image in Workspaces) but if you work-around that with `unset DB_HOST`, then it should work (and that's the part we're changing here)

Reply:
Let's try to not start another "HOST" confusion again :smile:

Reply:
Even without this change, if you try to run `yarn db-test`, it will fail in exactly the same way.

Reply:
Which is sad, and I fixed it like 2 times already, but somehow we keep fix-breaking things in circles :grimacing: